permissions/plugin-cisco-spark-notifier.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "cisco-spark-notifier"
+paths:
+- "org/jenkins-ci/plugins/cisco-spark-notifier"
```

Comment:
Woops sorry about that, fixed it here:
https://github.com/jenkinsci/cisco-spark-notifier-plugin/blob/master/build.gradle#L32